### PR TITLE
Provide a default safe tempest profile definition by default

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: ci_framework/roles/tempest/README.md [hunk 1]
```diff
@@ -15,24 +15,29 @@ become - Required to install required rpm packages
 * `cifmw_tempest_remove_container`: (Boolean) Cleanup tempest container after it is done. Default to `false`
 * `cifmw_tempest_tests_skipped`: (List) List of tests to be skipped. Setting this will not use the `list_skipped` plugin
 * `cifmw_tempest_tests_allowed`: (List) List of tests to be executed. Setting this will not use the `list_allowed` plugin
-* `cifmw_tempest_tempestconf_profile`: (Dictionary) List of settings to be overwritten in tempest.conf.
+* `cifmw_tempest_tempestconf_profile_default`: (Dictionary) Default tempest.conf to be used.
```
Comment (cjeanner) [inline ci_framework/roles/tempest/README.md:18]: I wouldn't expose it

File: ci_framework/roles/tempest/README.md [hunk 2]
```diff
@@ -15,24 +15,29 @@ become - Required to install required rpm packages
...
+* `cifmw_tempest_tempestconf_profile_overrides`: (Dictionary) Overrides of the `cifmw_tempest_tempestconf_profile_default` vars.
```
Comment (cjeanner) [inline ci_framework/roles/tempest/README.md:19]: I'd keep the same old name (retro-compatibility)

File: ci_framework/roles/tempest/defaults/main.yml [hunk 3]
```diff
@@ -29,3 +29,6 @@ cifmw_tempest_remove_container: false
 cifmw_tempest_concurrency: 4
 cifmw_tempest_dns_servers:
   - "192.168.122.10"
+cifmw_tempest_tempestconf_profile_default:
```
Comment (cjeanner) [inline ci_framework/roles/tempest/defaults/main.yml:32]: I'd push that in `vars/main.yml` instead.

**Comment by pablintino** [comment]
@cjeanner all addressed

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**